src/core/runQuery.ts

```diff
@@ -32,7 +32,19 @@ export interface QueryOptions {
  formatResponse?: Function;
 }
 
+// Make the global Promise constructor Fiber-aware.
```

Comment:
Is it possible to move these four lines either to the meteor-integration package or Meteor itself?